### PR TITLE
Make ReviewGAP faster

### PR DESCRIPTION
**Pull request by mozillazg**
Skip unnecessary steps.

### CODE REVIEW

**Comment by aysylu** [comment]
@ooq and @wkozlik: please review :)

File: pkg/kritis/review/review.go [hunk 1]
```diff
@@ -159,6 +160,22 @@ func (r Reviewer) findUnsatisfiedAuths(image string, auths []v1beta1.Attestation
 	return notAttestedBy, attestedByAny
 }
 
+func (r Reviewer) hasSatisfiedAuth(image string, auths []v1beta1.AttestationAuthority, c metadata.ReadOnlyClient) bool {
```
Comment (wkozlik) [inline pkg/kritis/review/review.go]: Could you add a comment about the purpose of this function. Otherwise LGTM

Comment (mozillazg) [reply pkg/kritis/review/review.go]: Thanks for your review, a comment has been added.

**Comment by ooq** [comment]
Thanks for the PR!
This will require additional after #455 is merged (which I will do today), just a heads up.

**Comment by ooq** [comment]
@mozillazg can you update the PR with master? thanks!

**Comment by mozillazg** [comment]
@ooq After #455 was merged, this PR is no longer needed.

https://github.com/grafeas/kritis/blob/fea1bd99fab40ce1602a1a093cd718008f31180a/pkg/kritis/review/review.go#L144-L152

**Comment by ooq** [comment]
@mozillazg got it. Thanks for checking this!

**Pull request closed without merging**